willeswa/politico-ui
# 164276000 signin UI-API link

#### What does this PR do?
Implements the fetch api on the signin page to allow user login
#### Description of Task to be completed?
Fetch the API designed earlier using JavaScript Vanilla, HTML and CSS
#### How should this be manually tested?
On `https://willeswa.github.io/politico-app/UI/signin.html` try signing up for a new account. You should be able to send a post request to the server and create a new account
#### Any background context you want to provide?
Previously, the front end design was not linked to the api and requests could only happen via postman
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164276000
